src/documents/management/commands/document_exporter.py

```diff
@@ -71,6 +73,11 @@ def dump(self):
                 t = int(time.mktime(document.created.timetuple()))
                 os.utime(target, times=(t, t))
 
+            with open(thumbnail_target, "wb") as f:
+                f.write(GnuPG.decrypted(document.thumbnail_file))
+                t = int(time.mktime(document.created.timetuple()))
```

Comment:
This line is redundant and could probably be moved just above line 71.  That way you can remove this line and line 73.